addons/mail/static/src/core/common/message.scss

```diff
@@ -51,6 +51,11 @@
     &:not(.o-note) {
         padding-left: (map-get($spacers, 2) + map-get($spacers, 3)) / 2;
         padding-right: (map-get($spacers, 2) + map-get($spacers, 3)) / 2;
+
+        .o-mail-Message.o-editing &:not(.o-note) {
```

Comment:
It's already inside `&:not(.o-note)`, it's necessary for specificity or something?

Reply:
I can remove it indeed, I was focused too much on the BEM classname at hand!